src/test/name_applications_tests.cpp

```diff
@@ -128,4 +128,14 @@ BOOST_AUTO_TEST_CASE( valid_json )
     BOOST_CHECK_EQUAL(IsValidJSONOrEmptyString(""), true);
 }
 
+BOOST_AUTO_TEST_CASE( minimal_json){
+    BOOST_CHECK_EQUAL(IsMinimalJSONOrEmptyString("{\"bar\":[1,2,3]}"), true);
+
```

Comment:
IMHO there is no need for the blank lines between the checks, unless you prefer that for some reason.

Reply:
I think it looks better that way, fits the style of the rest of the file IMO.